lib/exp_operations.js

```diff
@@ -19,20 +19,27 @@
 /**
  * @module aerospike/exp_operations
  *
- * @description This module provides functions to easily define operations to
- * be performed on a record via the {@link Client#operate} command.
+ * @description This module provides functions to easily create read/write operations
+ * using Aerospike expressions to be performed on a record via 
+ * the {@link Client#operate} command.
  *
  * @see {@link Client#operate}
+ * @see {@link Client#exp}
  *
  * @example
  *
  * const Aerospike = require('aerospike')
  * const op = Aerospike.operations
+ * const exp = Aerospike.expressions
  * const key = new Aerospike.Key('test', 'demo', 'mykey1')
- *
+ * const tempBin = 'ExpVar'
```

Comment:
This constant `tempBin` is used exactly once in this example - I'm not sure that it helps with readability, as it's not clear what its purpose is.